kata-deploy/k8s-1.13/runtimeclass-crd.yaml

```diff
@@ -0,0 +1,26 @@
+kind: CustomResourceDefinition
+apiVersion: apiextensions.k8s.io/v1beta1
+metadata:
+  name: runtimeclasses.node.k8s.io
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  group: node.k8s.io
+  version: v1alpha1
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+  names:
+    plural:   runtimeclasses
+    singular: runtimeclass
+    kind:     RuntimeClass
+  scope: Cluster
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            runtimeHandler:
+              type: string
+              pattern: '^([a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*)?$'
```

Comment:
Can a handler *really* start with a number?

Reply:
apparently.  This was pulled from K8S - I don't maintain the regex..

Reply:
Seems like it might be a bug, but let's be consistent with k8s I guess... ;)